### DISCUSSION TITLE
3rd-party "Tracked In" should be less prominent

### DISCUSSION THREAD

**Discussion by Gaelan** [community/community#4306] [Projects and Issues]
Hey, I was looking at https://github.com/rust-lang/cargo/issues/5034 today, and it now has a very prominent message saying "Tracked In EmbarkStudios/rust-ecosystem#22". As far as I can tell, that Embark repo is some game studio's effort to track Rust features that would make Rust more suitable for their usage. While that information is probably worth displaying *somewhere*, it seems far less important than, say, the issue's open status and its title(!), so it's weird that it's displayed as prominently as it is.

Also, I'd be curious to see what happens with popular issues (say, a long-standing bug in a popular framework) that end up "tracked in" many downstream projects.

Maybe issues being tracked from another organization should be listed in the sidebar on the right, and only ones from the same organization should appear in the header?

<img width="963" alt="Screen Shot 2021-06-24 at 12 16 47 PM" src="https://user-images.githubusercontent.com/4495750/123320011-0e27fb80-d4e6-11eb-8aa6-98c8f22c0245.png">

**Comment by azenMatt** [comment]
Thanks for sharing this @Gaelan - I agree this feels like an unintended side-effect from what we were trying to capture. 

I like the idea of de emphasising this when it is outside the organisation.

**Comment by 0xdevalias** [comment]
Slightly off topic.. but what causes that 'tracked in' to show in the first place out of curisosity?
